### ISSUE TITLE
Post List: Clicking on like takes you to stats

### ISSUE THREAD

**Issue by gwwar** [Automattic/wp-calypso#19168]
1. Navigate to https://wordpress.com/posts
2. Select a site with posts
3. Attempt to click on the Star icon to like a post
Expected: we like a post
Actual: We're taken to a stats page eg: https://wordpress.com/stats/post/1514/gwwartest.wordpress.com

<img width="734" alt="screen shot 2017-10-25 at 3 58 05 pm" src="https://user-images.githubusercontent.com/1270189/32027374-88a8a520-b99d-11e7-9139-ffcf89853c44.png">

<img width="847" alt="screen shot 2017-10-25 at 3 58 20 pm" src="https://user-images.githubusercontent.com/1270189/32027380-902c3db6-b99d-11e7-8198-692bb1f03a89.png">

**Comment by gwwar** [comment]
If this was intended behavior feel free to close this out.

**Comment by lsinger** [comment]
IIRC I've heard that this was intended. @shaunandrews @mattsherman thoughts?

**Comment by mattsherman** [comment]
Since this is a list of the posts on the user's own site, I think that taking the user to the stats page is more valuable than letting the user like their own post.

On multi-author sites, especially private ones, there would be some value in liking a post on your own site, but I think would be confusing to provide this ability from this management screen.

**Comment by lsinger** [comment]
Thanks @mattsherman — I think we can close.

**Comment by samouri** [comment]
its a bit strange from a consistency point of view. In other Calypso pages you are capable of liking your own post with the same exact star button

**Comment by shaunandrews** [comment]
Um, I don't know that this is intended behavior — it feels really, really strange. Especially considering that the back button on the post stats takes you back to `/stats/` and not `/posts/`

**Comment by mattsherman** [comment]
I agree that the back button behavior feels wrong. I still hold that liking posts on your own site from this screen doesn't feel right either though. I'm not sure what the original intention on the design was though, or if/when it was changed at some point.

**Comment by shaunandrews** [comment]
> In other Calypso pages you are capable of liking your own post with the same exact star button

This is the crux of it. Having the star icon take you somewhere (which is super disruptive, and with no easy way back) breaks the pattern (and trust) of how the like button works throughout Calypso and WordPress.com. It may seem silly to like your own post (though, I've done it lots) but I see no harm in it. In fact, I bet there's some random use-case where liking your own post helps you do "something."

Its also repetitive. There's a Stats button (icon/label) down with the other actions at the bottom of the Post card.

**Comment by ianstewart** [comment]
I'll +1 Shaun here. It seems very strange to shoot back to stats when you try to like your own post.

**Comment by mattsherman** [comment]
Agreed -- I've come around on this one.
